src/go/k8s/tests/e2e/update-conf-image/00-assert.yaml

```diff
@@ -6,4 +6,5 @@ metadata:
 status:
   readyReplicas: 1
   replicas: 1 # Temporarily, the number of replicas becomes 2. We wait until the new version is up and running alone - or set maxSurge=0 in deployment.
+  availableReplicas: 1
```

Comment:
I think the best option for now would be:
> updatedReplicas (int32)
> Total number of non-terminated pods targeted by this deployment that have the desired template spec.

Ref https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentStatus

Reply:
Added. Rerunning test.